stdlib/_thread.pyi

```diff
@@ -16,11 +16,16 @@ class LockType:
     def acquire(self, blocking: bool = True, timeout: float = -1) -> bool: ...
     def release(self) -> None: ...
     def locked(self) -> bool: ...
+    def acquire_lock(self, blocking: bool = True, timeout: float = -1) -> bool: ...
+    def release_lock(self) -> None: ...
+    def locked_lock(self) -> bool: ...
     def __enter__(self) -> bool: ...
     def __exit__(
         self, type: type[BaseException] | None, value: BaseException | None, traceback: TracebackType | None
     ) -> None: ...
 
+lock: TypeAlias = LockType  # noqa: Y042
```

Comment:
This seems to be new in Python 3.13. It's also a regular alias, not a type alias:

```suggestion
if sys.version_info >= (3, 13):
    lock = LockType
```